.github/workflows/publish-release.yml

```diff
@@ -35,10 +35,13 @@ jobs:
       - uses: MetaMask/action-publish-release@v2.0.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Install
-        run: |
-          yarn install
-          yarn build
+      - name: Cache yarn dependencies
+        uses: actions/cache@v2
```

Comment:
This should probably be `actions/cache@v3`, since that's what we use below too.

Reply:
Done. I had intentionally made it identical to to the build-lint-test steps, but those have now been updated as well.